docs/javascript-installation.md

```diff
@@ -260,6 +260,8 @@ This is a required setting. If not explicitly set, the agent will try to automat
 
 Example value: `974c3566eb8e221d130db86a7ce1f99703fe2e69`
 
+> This parameter may also be configured in `scope.yml` through the key `commit_sha`.
+
```

Comment:
```suggestion
```

Because the commit info is highly dynamic, I wouldn't recommend setting it in the yaml file

Reply:
ok I'll also update the specs